7.7/hooks/post_push

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+VERSION=$(cat Dockerfile | grep '^ENV VERSION' | cut -d" " -f3)
+TAGS=($VERSION 7)
```

Comment:
Can you please remove the 7 from the previous “latest” 7 folder?

Reply:
Sure; should be done now.

Reply:
👍